### DISCUSSION TITLE
Qutebrowser doesn't start pulseaudio automatically

### DISCUSSION THREAD

**Discussion by dag-h** [qutebrowser/qutebrowser#5744] [Q&A]
**Version info**:
qutebrowser v1.13.1
Git commit: 
Backend: QtWebEngine (Chromium 80.0.3987.163)
Qt: 5.15.1 (compiled 5.15.0)

**Does the bug happen if you start with `--temp-basedir`?**:
Yes

**Description**
I am using Void Linux, and [it's recommended you let pulseaudio start automatically](https://docs.voidlinux.org/config/media/pulseaudio.html) by the programs that use it. This works when using Firefox, upon logging in and opening Firefox I do have sound. If I log in and open Qutebrowser, sound does not work. If I open Qutebrowser after I have started Firefox once it continues to work. I'm not sure if Qutebrowser has anything to do with this, or if this is a Qt issue.

**How to reproduce**
* Install Void Linux
* Install PulseAudio
* Install Qutebrowser
* Start Qutebrowser

**Comment by The-Compiler** [comment]
Can you check if you see the same with KDE Falkon and/or with Chromium?

**Comment by dag-h** [comment]
Somehow it fixed itself. This was a fresh install so it might have gotten fixed by some package I installed. Sadly I won't be able to pinpoint exactly what it was.

**Comment by dag-h** [comment]
Never mind, problem still exists. Seeing the same issue when using Falkon or Chromium. Firefox works as it should, starting pulseaudio when needed.

**Comment by dag-h** [comment]
To add, I'm also seeing the same issue with Discord, doesn't detect my devices, if that makes any difference. Starting Pavucontrol fixes the issue for all applications.

**Comment by The-Compiler** [comment]
Clearly this isn't a qutebrowser issue then. Thus, I'm converting this to a discussion.
